source/solvers/postprocessing_cfd.cc

```diff
@@ -159,6 +159,137 @@ calculate_pressure_drop<3, TrilinosWrappers::MPI::BlockVector>(
   const unsigned int                        inlet_boundary_id,
   const unsigned int                        outlet_boundary_id);
 
+template <int dim, typename VectorType>
+double
+calculate_total_pressure_drop(
```

Comment:
Maybe this calculation could have been merged with the regular calculate pressure drop? since any both are combined